src/main/java/com/zuniorteam/racingcar/core/strategy/RandomMovingStrategy.java

```diff
@@ -0,0 +1,14 @@
+package com.zuniorteam.racingcar.core.strategy;
+
+import java.util.Random;
+
+public class RandomMovingStrategy implements MovingStrategy {
+
+    private static final int RANDOM_SIZE = 10;
+    private static final int FLAG_NUMBER = 4;
+
+    @Override
+    public boolean isMoving() {
```

Comment:
isMoving()보단
```suggestion
    public boolean canMoving() {
```
이 적절할 거 같아요!

Reply:
헉, `isMoving()`은 확실히 이상하네요 ㅎㅎ... 감사합니다!!